api/services/ocean/spark/schemas/oceanSparkWorkspaceCreateWrapper.yaml

```diff
@@ -0,0 +1,6 @@
+type: object
+required:
+  - workspace
```

Comment:
not sure about this wrapper.
IIRC the body should contain only `name` field, there is no need for a wrapper 

my postman request when I tested the optimizer:
<img width="746" alt="Screenshot 2024-04-10 at 18 23 15" src="https://github.com/spotinst/openapi/assets/21658174/f87e5bd8-ea3e-4d2c-a5ae-20f7e52325e2">